src/llmcompressor/observers/base.py

```diff
@@ -22,9 +24,14 @@ class Observer(Module, RegistryMixin):
     pair
     """
 
-    def __init__(self, quantization_args: QuantizationArgs):
+    def __init__(
+        self,
+        quantization_args: QuantizationArgs,
+        global_scale: Optional[torch.Tensor] = None,
+    ):
         self.quantization_args: QuantizationArgs = quantization_args
         super().__init__()
+        self.global_scale: torch.Tensor = global_scale
```

Comment:
you're setting an `Optional[torch.Tensor]` as `torch.Tensor` here, don't we need a check to confirm it's not None?

Reply:
+1

Reply:
The global scale is only used if it is not None, in `calculate_qparams`

Reply:
in [this line](https://github.com/neuralmagic/compressed-tensors/blob/a6327c79601e81bcf4ffda69d31d3af5e8e65a5c/src/compressed_tensors/quantization/utils/helpers.py#L71), it has the right typehint that it's `Optional[torch.Tensor]`. Why not have the same type here?